round4/liuxk.md

```diff
@@ -0,0 +1,50 @@
+# Chrome Extension Idea: Web Styler
+
+## Authors
+
+Group 12: Kevin Liu, Adam Gorka, Andy Li, Judah Nouriyelian
+
+## Problem Statement
+
+By default, in order for users to change visual elements of webpages, they must manually inspect the underlying HTML/CSS, locate the desired elements, and change their states. This process is often time-consuming, if not difficult or even impossible for inexperienced or visually impaired users. We propose this extension as a way for users to customize these elements more easily.
+
+## Target Audience
+
+Targeted towards visually impaired individuals, specifically for colorblind and near-blind. Our extension will adjust the style of a website so that any text is larger. In addition, any coloring/style will be adjusted so that it's more clearly visible for individuals with color blindness.
+Many individuals that are color blind or visually impaired struggle with viewing web pages that are not formatted well. They would significantly benefit from a tool that enlarges text and changes colors to be color-blind friendly. 
+Additionally, the extension can also be used by other people who simply want an easy way to change such features for their own personal preferences.
+
+## Description
+
+This extension allows users to change visual features (e.g., text size, color) on websites to suit their preferences. Giving users a way to interact with these visual elements more directly and easily, the tool can be used for a variety of purposes, from accessibility to personalization.
+
+## Selling Points
+
+1. Enhanced accessibility for everyone.
+2. Customizable styling. Users can change the size of fonts as well as the color/filters applied.
+3. Convenience. Chrome extension will be applied automatically to every webpage.
+4. Compatibility. Will be compatible with any type of website.
+5. Personalization. Users can create and save certain themes/styles as favorites.
+
+## User Stories
+
+_[List user stories that describe the main features of your Chrome Extension. Use the following template: "As a [user role], I want to [goal] so that [benefit]." Fill in the brackets with the appropriate information. For Round 1, write 5 user stories; Round 2, write 10 user stories; Round 3, write 15 user stories.]_
```

Comment:
@liuxk83 Don't forget to remove the instructions from your final submission, this should be removed:
```
_[List user stories that describe the main features of your Chrome Extension. Use the following template: "As a [user role], I want to [goal] so that [benefit]." Fill in the brackets with the appropriate information. For Round 1, write 5 user stories; Round 2, write 10 user stories; Round 3, write 15 user stories.]_
```